pkg/sql/sem/tree/schema_helpers_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package tree_test
+
+import (
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/sql/parser"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
+)
+
+func TestIsAllowedLDRSchemaChange(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	for _, tc := range []struct {
+		stmt      string
+		isAllowed bool
+	}{
+		{
+			stmt:      "CREATE INDEX idx ON t (a)",
+			isAllowed: true,
+		},
+		{
+			stmt:      "DROP INDEX idx",
+			isAllowed: true,
+		},
+		{
+			stmt:      "ALTER TABLE t ADD COLUMN a INT",
+			isAllowed: true,
```

Comment:
i think this test needs to be updated in this commit, as add nullable column is not allowed yet

Reply:
done